### PR TITLE
MAINT-26336 : Security : any user can post a News in any space

### PR DESCRIPTION
**Pull request by lbengayed**

### CODE REVIEW

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java [hunk 1]
```diff
@@ -1188,6 +1188,7 @@ public void unarchiveNews(String newsId) throws Exception {
   }
 
   /**
+   * Check if the current user can create a news
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/news/NewsServiceImpl.java:1191]: Unneded change I think

File: services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java [hunk 2]
```diff
@@ -729,10 +729,10 @@ public Response deleteNews(@Context HttpServletRequest request,
   @RolesAllowed("users")
   @ApiOperation(value = "check if the current user can create a news in the given space", httpMethod = "GET", response = Response.class, notes = "This checks if the current user can create a news in the given space", consumes = "application/json")
   @ApiResponses(value = { @ApiResponse(code = 200, message = "News deleted"),
-      @ApiResponse(code = 400, message = "Invalid query input"),
-      @ApiResponse(code = 401, message = "User not authorized to check"),
-      @ApiResponse(code = 404, message = "Space not found"),
-      @ApiResponse(code = 500, message = "Internal server error") })
+          @ApiResponse(code = 400, message = "Invalid query input"),
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java:732]: Unneeded change

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java (hunk 1 above)
Comment (lbengayed) [reply services/src/main/java/org/exoplatform/news/NewsServiceImpl.java]: done!

File: services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java (hunk 2 above)
Comment (lbengayed) [reply services/src/main/java/org/exoplatform/news/rest/NewsRestResourcesV1.java]: done!

File: services/src/main/java/org/exoplatform/news/NewsServiceImpl.java [hunk 3]
```diff
@@ -1199,7 +1199,6 @@ public boolean canCreateNews(String authenticatedUser, Space space) throws Excep
     Boolean spaceHasRedactors = false;
     Boolean isCurrentUserRedactor = false;
     String spaceGroupId = space.getGroupId();
-
```
Comment (ahamdi) [inline services/src/main/java/org/exoplatform/news/NewsServiceImpl.java:1202]: Unneded change !

**Pull request merged**